ts/parts/Utilities.ts

```diff
@@ -2641,6 +2650,32 @@ const getStyle = H.getStyle = function (
     return style;
 };
 
+/**
+ * Get the defer as a number value from series animation options.
+ *
+ * @function Highcharts.getDeferTime
+ *
+ * @param {Highcharts.Chart} chart
+ *        The chart instance.
+ *
+ * @return {number}
+ *        The numeric value.
+ */
+const getDeferTime = H.getDeferTime = function (chart): number {
+
+    const seriesDelay = chart.series.map(function (series): number {
+        var seriesAnim = series.options.animation as any,
+            seriesDuration = seriesAnim.duration;
```

Comment:
Should be `seriesAnim && seriesAnim.duration`. Otherwise can throw errors that you check below (`if (seriesAnim)`).